cypress/e2e/tests/migration/applicationinventory/analysis/pythonAnalysis.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+} from "../../../../../utils/utils";
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import { Application } from "../../../../models/migration/applicationinventory/application";
+
+let applicationsList: Array<Analysis> = [];
+describe(["@tier2"], "Source Analysis", () => {
+    before("Login", function () {
+        login();
+        cy.visit("/");
+    });
+
+    beforeEach("Load data", function () {
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+
+        // Interceptors
+        cy.intercept("POST", "/hub/application*").as("postApplication");
+        cy.intercept("GET", "/hub/application*").as("getApplication");
+        cy.intercept("DELETE", "/hub/application*").as("deleteApplication");
+        cy.visit("/");
+    });
+    it(["@tier1"], "Source analysis on python app", function () {
+        const application = new Analysis(
+            getRandomApplicationData("pythonApp_Source", {
+                sourceData: this.appData["python-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["source_analysis_on_pythonApp"])
+        );
+        Application.open();
+        application.create();
+        applicationsList.push(application);
+        cy.wait("@getApplication");
+        application.analyze();
+        application.verifyAnalysisStatus("Completed");
+        application.verifyEffort(this.analysisData["source_analysis_on_pythonApp"]["effort"]);
+        application.verifyMergedLogContain();
+    });
+    after("Perform test data clean up", () => {
```

Comment:
Let's also assert that the two rules were fired.

Reply:
Sure, Currently we are blocked due to this bug https://issues.redhat.com/browse/MTA-4885, I don't have relevant info to add in the issues for analysis.json, therefore we can add this once issue is fixed.